tests/fstab_test.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
The test needs to be placed either in `storage_tests` (if it uses "real" storage) or in `unit_tests` (if it doesn't require root privileges). Ideally this functionality would be covered in both test suites.

Reply:
The tests are now placed in `storage_tests` and `unit_tests` .